apps/hls_examples/stereo_hls/run_zynq.cpp

```diff
@@ -8,38 +8,33 @@
 #include "pipeline_zynq.h"
 #include "pipeline_native.h"
 
-#include "benchmark.h"
-#include "halide_image.h"
+#include "BufferMinimal.h"
 #include "halide_image_io.h"
 
 using namespace Halide::Tools;
+using namespace Halide::Runtime::HLS;
+
+extern "C" {
+extern int halide_zynq_init();
+}
 
 int main(int argc, char **argv) {
     if (argc < 5) {
         printf("Usage: ./run left.png left-remap.png right0224.png right-remap.png\n");
         return 0;
     }
-    // Open the buffer allocation device
-    int cma = open("/dev/cmabuffer0", O_RDWR);
-    if(cma == -1){
-        printf("Failed to open cma provider!\n");
-        return(0);
-    }
+    // Initialize zynq runtime
+    halide_zynq_init();
 
-    // open the hardware
-    int hwacc = open("/dev/hwacc0", O_RDWR);
-    if(hwacc == -1) {
-        printf("Failed to open hardware device!\n");
-        return(0);
-    }
+    // Halide::Buffer<T> cannot be automatically converted to halide_buffer_t*
```

Comment:
I think Buffer can be converted to halide_buffer_t * automatically.
https://github.com/jingpu/Halide-HLS/blob/HLS/src/runtime/HalideBuffer.h#L885

Reply:
My bad... I was using Halide::Buffer<T>, instead of Halide::Runtime::Buffer<T>.

Reply:
remove the comment?